### DISCUSSION TITLE
Mix icons for checkbox from two sets [Sep 23]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3885] [Discord Threads]
> 🙋‍♂️ **vikiiingur**  
## In a checkbox-boolean value, is it possible to have dual settings for icons+colours? For example, 0=checkmark in green, 1=crossmark in red  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1020276365305577543/unknown.png)

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### No. only pre-configured icons supported  
#  
> 🙋‍♂️ **vikiiingur**  
#### can it be added as a feature request maybe?  
#  
> 🙋‍♂️ **vikiiingur**  
#### of course, my scenario can be only an optional and used when defined, otherwise the current functionality would be applicable  
#  
> 🙋‍♂️ **vikiiingur**  
#### thanks!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1020276365821497406)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
